### PR TITLE
[PT] Translation for Debugging page

### PR DESCRIPTION
**Pull request by rrmartins**
Issue https://github.com/doomspork/elixir-school/issues/355 

Translation of the Debugging page into Portuguese

Version in English [here](https://elixirschool.com/lessons/specifics/debugging/)

### CODE REVIEW

File: pt/lessons/specifics/debugging.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+# Dialyxir e Dialyzer
+
+O [Dialyzer](http://erlang.org/doc/man/dialyzer.html) (**DI**screpancy **A**nal**YZ**er for **ER**lang), é uma ferramenta para análise de
+código estático. Em outras palavras eles _leem_ e analisam mas não _rodam_ o código. Exemplo: procurando por alguns bugs, códigos mortos,
+desnecessários ou inacessíveis.
+
+O [Dialyxir](https://github.com/jeremyjh/dialyxir) é uma tarefa mix para simplificar o uso do Dialyzer em Elixir.
+
+Ferramentas de especificação como o Dialyzer ajuda a enteder melhor o código. Este é um exemplo do que é legível e compreensível apenas para outros seres humanos (se apenas existe e é bom escrito), `@spec` usar sintaxe mais formal e que pode ser entendido pela máquina.
```
Comment (pragmaticivan) [inline pt/lessons/specifics/debugging.md]: what about `ajudam` instead of `ajuda` ?

Comment (pragmaticivan) [inline pt/lessons/specifics/debugging.md]: `bom escrito`

File: pt/lessons/specifics/debugging.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+![Debugger Screenshot 1]({{ site.url }}/assets/debugger_1.png)
+
+Depois de ter anexado o nosso módulo para o depurador estará disponível no menu à esquerda:
+
+![Debugger Screenshot 2]({{ site.url }}/assets/debugger_2.png)
+
+## Criando breakpoints
+
+Um breakpoint um ponto no código onde a execução será interrompida. Temos duas maneiras de criar breakpoints:
```
Comment (pragmaticivan) [inline pt/lessons/specifics/debugging.md]: `Um breakpoint é um ponto`

**Comment by doomspork** [comment]
Thank you @rrmartins and @pragmaticivan!!

**Comment by pragmaticivan** [comment]
ping @rrmartins

**Comment by rrmartins** [comment]
@pragmaticivan sorry. Thanks for the review.

**Comment by rrmartins** [comment]
Who makes the merge and closes the PR?

**Comment by pragmaticivan** [comment]
Thanks for the contribution @rrmartins !

**Pull request merged**